src/components/pages/projects.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled from "styled-components";
+import DefaultTemplate from "../templates/DefaultTemplate";
+import Card from "../molecules/Card";
+
+const GroupCard = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const mockProjects = [
+  {
+    name: "Project #1",
+    description: "It's my first",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first project",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first project",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first project",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first project",
+    createdAt: "22 days ago",
+  },
+  {
+    name: "Project #1",
+    description: "It's my first project",
+    createdAt: "22 days ago",
+  },
+];
+
+const ProjectsPage = () => {
+  const groupProjectCard = mockProjects.map((project) => (
+    <Card name={project.name} description={project.description} createdAt={project.createdAt} />
```

Comment:
You need to add special `key` prop to avoid error in console.
You can add an `id` in projects and use it
```
<Card key={project.id} name={project.name} description={project.description} createdAt={project.createdAt} />
```
or you can add `uuid` lib in project
```
import { v4 } from "uuid";

<Card key={`table-row-${v4()}`} name={project.name} description={project.description} createdAt={project.createdAt} />
```
<img width="809" alt="image" src="https://user-images.githubusercontent.com/79587920/227775381-fee28a6d-fa1d-4b25-866f-dae7ff302db7.png">